function/src/index.py

```diff
@@ -146,8 +158,8 @@ def get_cert_info(certificate):
   Serial Number: {cert.serial_number}
   Issuer: {cert.issuer.rfc4514_string()}
   Validity:
-    Not Before: {cert.not_valid_before}
-    Not After: {cert.not_valid_after}
+    Not Before: {cert.not_valid_before_utc}
+    Not After: {cert.not_valid_after_utc}
```

Comment:
This change was because of a warning that appeared in the log when I was testing. The plain `not_valid_before` and `not_valid_after` are deprecated. Upstream calls them "naive" timestamps.

Reply:
Got it. Thats right. Thanks